### ISSUE TITLE
String is too large to store, 401 error

### ISSUE THREAD

**Issue by icidasset** [blockstack/blockstack-core#499]
From the API logs:

```
[2017-07-19 20:19:32,020] [ERROR] [rpc:228] (98062.123145570938880) Request from 127.0.0.1:54920 is too long (2776837 >= 1048576)
[2017-07-19 20:19:32,031] [ERROR] [rpc:252] (98062.123145570938880) Failed to read request
127.0.0.1 - - [19/Jul/2017 20:19:32] "PUT /v1/stores/ ... /files?path=/ongaku-ryoho-v1_0.json& ... HTTP/1.1" 401
```

I'm doing the following using Blockstack.js:

```js
const content = JSON.stringify(dataContainer);
blockstack.putFile(BLOCKSTACK_FILE_PATH, content);
```

Content has a length of `2081126` characters.

**Comment by kantai** [comment]
Thanks for finding this! 

The plan will be to increase this limit for the storage endpoint because it isn't a public endpoint, but keep the limit in place for other API calls. Will work on this now.

**Comment by kantai** [comment]
Fixed in a672a9dcb7881835d4334fe45c912bbad00e12a2 (should ship with next browser release).

**Comment by icidasset** [comment]
Spoke too soon, this isn't in the current browser release right? (ie. `v0.11.2`)

**Comment by icidasset** [comment]
Fixed in browser `v0.12.0`, thanks again!
